pytorch_tabnet/tab_network.py

```diff
@@ -40,6 +40,7 @@ def forward(self, x):
 
 class TabNetNoEmbeddings(torch.nn.Module):
```

Comment:
As the name suggest, this class is supposed to be basic tabnet with no embeddings.

Reply:
Yup and I agree it's a nice distinction to keep! So my thought was to add an optional parameter (`feature_embed_widths`) to this class where, if they want, the user can tell TabNet to treat multiple columns of the input as a single "feature" for attention purposes. By default (`None`) `TabNetNoEmbeddings` should work as before, treating every column independently. There are a couple of ways this API could pass in the information required, so the current one is a list of how wide each "feature" in your input is: E.g. `[1, 1, 2, 3]` would mean:

- `input_dim` is 1+1+2+3=7
- `n_features` is 4
- First two columns are scalar features, next two are a feature with emb_dim 2, next three are a feature with emb_dim 3